crates/pipeline-manager/src/api/api_key.rs

```diff
@@ -29,84 +29,60 @@ pub(crate) struct NewApiKeyRequest {
 /// Response to a successful API key creation.
 #[derive(Debug, Serialize, ToSchema)]
 pub(crate) struct NewApiKeyResponse {
-    /// Id of the newly created API key.
-    #[schema(example = 42)]
-    api_key_id: ApiKeyId,
+    /// Identifier of the newly created API key.
+    #[schema(example = "00000000-0000-0000-0000-000000000000")]
```

Comment:
Something I wondered about when writing the CLI, is there a good reason the user ever needs to know the UUID for the key?